### DISCUSSION TITLE
Shall we manage adopters directly in the landscape project?

### DISCUSSION THREAD

**Discussion by benjagm** [json-schema-org/community#710] [General]
### Context

I'd like to bring up a topic for discussion regarding the management of adopters within our organization. Some months ago we launched the [adopters.md](https://github.com/json-schema-org/community/blob/main/ADOPTERS.md) to highlight JSON Schema adopters. However, with the recent launch of the [JSON Schema landscape project](https://landscape.json-schema.org/), I believe we're facing a certain level of duplicity that requires discussion and alignment.

The landscape project provides a comprehensive and visual overview of the JSON Schema ecosystem, including adopters. Given its broader scope, enhanced visual representation and user experience, I propose that we consider managing adopters directly within the landscape project instead of maintaining a separate list in the adopters.md file.

By consolidating adopter management within the landscape project, we can streamline our efforts and avoid redundancy. Additionally, it offers a more accessible and engaging way for both contributors and users to explore JSON Schema adopters within the broader context of the ecosystem.

### Proposal
I suggest that we update the adopters.md file to include a note directing visitors to the JSON Schema landscape as the preferred platform for viewing and managing adopters. This way, we can ensure consistency and provide clear guidance to our community members.

In addition, I think we should consider making it easier to add a new adopter, by allowing to anyone adding a new adopter if there is public documentation supporting it, without requiring explicit confirmation from the organization. The same way we'll  enable a process to make it easy the removal of an organization by organization representatives.

We'd love to get feedback from the Community to make sure we take the best decision.

**Comment by jviotti** [comment]
Shall we add all companies from https://github.com/sourcemeta/awesome-jsonschema?tab=readme-ov-file#adoption to the landscape? Then we can remove that section from my `awesome-jsonschema` repo

**Comment by Relequestual** [comment]
I think this is a good direction. Visually it will be more impactful.

I am not a lawyer, but I believe if we are linking to articles or other evidence, use of company logos would fall under "news" fair use.

I think we should specify in the "guide" section that for Adopters, some have given permission and some have not given permission, and we will remove any entry if there is a valid legal request to remove it.

**Comment by DarhkVoyd** [comment]
This is a good idea. How would we approach situations where logos of some adapters/tooling are not available in svg format?
